modules/generic/src/main/scala/cats/xml/generic/Configuration.scala

```diff
@@ -26,13 +30,15 @@ object Configuration {
   val default: Configuration = Configuration(
     useDefaults          = false,
     unwrapValueClasses   = true,
-    discriminatorAttrKey = None
+    discriminatorAttrKey = None,
+    useLabelsForNodes    = false
   )
 
   implicit val showConfig: Show[Configuration] = Show.show(c => s"""
       |Configuration(
-      | useDefaults        = ${c.useDefaults},
-      | unwrapValueClasses = ${c.unwrapValueClasses},
-      | discriminatorAttrKey  = ${c.discriminatorAttrKey}
+      | useDefaults          = ${c.useDefaults},
+      | unwrapValueClasses   = ${c.unwrapValueClasses},
+      | discriminatorAttrKey = ${c.discriminatorAttrKey},
+      | useLabelsForNodes    = ${c.useLabelsForNodes}
```

Comment:
Maybe this isn't a configuration thing. Maybe this is a `XmlTypeInterpreter` thing. Or maybe they're both things and this provides the default to that... 🤷